src/config/tomlconfig.rs

```diff
@@ -11,6 +11,8 @@ pub struct TomlConfig {
     pub description: Option<String>,
 
     pub output: Option<TomlOutputConfig>,
+
+    pub use_ace: Option<bool>,
```

Comment:
Yeah, I was thinking of doing something like this but wasn't quite sure which direction to go in.

So we would have a playpen-properties header with various options, and to use ace, the `.toml` would look like this:

> [playpen-properties]
> editable = true

Am I following you right?